lib/routes.js

```diff
@@ -54,7 +54,7 @@ const responseSchema = {
 async function defaultErrorHandler (err, request, reply) {
   if (err.data) {
     reply.code(200)
-  } else {
+  } else if (err.statusCode) {
```

Comment:
Could you add a unit test? I would also make it explicit and set the status code either to `200` or `500`. I'm not sure which one is correct.

Reply:
I opted in for 500, and added a test that represent exactly the error we had on our project.
It is a schema parsing error, but strangely, it is only thrown when running request, as opposed to schema syntax errors which are raised at startup.